### DISCUSSION TITLE
How to filter results with applications name?

### DISCUSSION THREAD

**Discussion by LilianJude** [microsoft-search/pnp-modern-search#3295] [Q&A]
Hello,

Is it possible to filter results with name of applications? I added "File Type" to search filters but only extension is displayed, how to have the same display as native search? 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/35682719/9e031156-36ab-4d12-a8d5-ab9a7406d7c9)

Thanks!

**Comment by wobba** [comment]
Custom development for a static file type filter is needed, which means someone have to donate their time to implement it.

**Reply by LilianJude** [reply]
Ok thanks! where can I start to complete this? Need to create spfx app ?

**Reply by wobba** [reply]
You would contribute to this github project, and requires knowledge about search, React, SPFx and connected web parts.
